index.html

```diff
@@ -122,10 +122,16 @@
       illuminate favorable or unfavorable consequences for their needs.
     </p>
     <p>
-      While this rubric allows the evaluation of many aspects of decentralization of
-      a DID Method, it is not exhaustive, and does not cover other factors that may
-      affect selection or adoption of a particular Method, such as privacy, security,
-      reliability, or efficiency.
+      This rubric is a collection of criteria for creating Evaluation
+      Reports that assist you, the Evaluator, in choosing a DID
+      Method.
+
+      While this rubric assists in the evaluation of many aspects of a
+      DID Method, it is not yet exhaustive.
```

Comment:
```suggestion
      DID Method, it is not exhaustive.
```
Remove the "yet" because it will never actually be exhaustive.